lib/core/managementClient.ts

```diff
@@ -621,22 +621,22 @@ export class ManagementClient extends LinkEntity {
           case descriptorCodes.sqlFilterList:
             if (Array.isArray(filtersRawData.value) && filtersRawData.value.length === 2) {
               rule.filter = {
-                expression: filtersRawData.value[0] && filtersRawData.value[0].value
+                expression: this._safelyGetTypedValue(filtersRawData.value[0])
```

Comment:
I wonder if we can use this method in other places in this file (not necessarily) -- see line 651 for a place where we might be able to use this method.

Reply:
Good catch. Yes, we can use the same there as well.